### ISSUE TITLE
VPN manager keeps asking password in terminal

### ISSUE THREAD

**Issue by koakenred** [Zomboided/service.vpn.manager#183]
Hi,

I have my own vpn server based on pivpn on my home network, because that way i can connect to my Freenas server where i store videos. When i wan't to use kodi outside my network i have to connect kodi through the vpn server to connect to the nas. I couldn't get it to work for a long time, but after a while i noticed that when i try to connect to my vpn server i get the following question in the putty terminal (that is connected to the raspberry pi with kodi):
![password question](https://user-images.githubusercontent.com/36958942/46814880-f7630a00-cd79-11e8-954f-9e8a22cbec46.PNG)
After entering the password here i was connected to the vpn. But i have to fill in the password in the terminal everytime i want to connect to the vpn this way. Is it possible to solve this and store the password in the VPN manager itself?

**Comment by Zomboided** [comment]
Duplicate of https://github.com/Zomboided/service.vpn.manager/issues/60

Right now your only option is not to have a password associated with the key

**Comment by Zomboided** [comment]
You might have some luck putting the password in a file and referencing that from the ovpn file that you offer up.  You won't get the ability to change/enter it in the Kodi environment though, at least not until I fix the bug this is a duplicate of.
